qt/scientific_interfaces/test/ISISReflectometry/MainWindow/MainWindowPresenterTest.h

```diff
@@ -396,13 +411,17 @@ class MainWindowPresenterTest : public CxxTest::TestSuite {
     auto presenter = MainWindowPresenterFriend(
         &m_view, &m_messageHandler, &m_fileHandler, std::move(encoder),
         std::move(decoder), std::move(slitCalculator),
-        std::move(makeBatchPresenter));
+        std::move(optionsDialogPresenter), std::move(makeBatchPresenter));
     return presenter;
   }
 
-  void verifyAndClear() {
+  void verifyAndClear(
+      std::unique_ptr<NiceMock<MockOptionsDialogPresenter>>
+          optionsDialogPresenter =
```

Comment:
If passing a raw pointer as per my other suggestion then it can have a default value of `nullptr` and we don't need to bother verifying it if it is null.